rendermodules/pointmatch/swap_point_match_collections.py

```diff
@@ -17,6 +19,81 @@
     "zValues": [1015]
 }
 
+def swap_pt_matches(render, source_collection, target_collection, z, match_owner=None):
+    session = requests.Session()
+    session.mount('http://', requests.adapters.HTTPAdapter(max_retries=5))
+
+    groupid = "{}.0".format(str(z))
+    match_owner = match_owner if match_owner is not None else render.DEFAULT_OWNER
```

Comment:
This doesn't need to change, but the logic should be handled by render-python -- owner in a call overrides the render default owner.

Reply:
yep this just adds a double check